admin.mss

```diff
@@ -220,10 +220,14 @@ overlapping borders correctly.
   text-dy: -10;
 }
 
-#nature-reserve-text[zoom >= 13][way_pixels > 192000] {
+#protected-areas-text[zoom >= 13][way_pixels > 192000] {
   text-name: "[name]";
   text-face-name: @book-fonts;
   text-fill: green;
+  [boundary='aboriginal_lands'],
+  [class='24'] {
```

Comment:
I guess aboriginal_lands already includes 24 case? Anyway class=24 alone sounds hard to understand.

Reply:
`boundary=aboriginal_lands` doesn't need `24`, but you're right that it's not very clear to only use `[class=24]` to capture the other possible tagging approach. I made this line more verbose, so the second selector is now `[boundary='protected_area'][protect_class='24']`